src/map/lua/luautils.h

```diff
@@ -63,6 +63,15 @@ extern sol::state lua;
 #include "lua_trigger_area.h"
 #include "lua_zone.h"
 
+enum SendToDBoxReturnCode : uint8
```

Comment:
If this doesn't need to be decay'd down into it's underlying type, make it an `enum class`